plugins/org.yakindu.sct.model.sruntime/src/org/yakindu/sct/model/sruntime/SRuntimeFactoryImplOverride.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2018 itemis AG - All rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * 
+ * Contributors:
+ * 	Andreas Muelder - itemis AG
+ * 
+ */
+package org.yakindu.sct.model.sruntime;
+
+import org.eclipse.emf.ecore.EDataType;
+import org.yakindu.sct.model.sruntime.impl.SRuntimeFactoryImpl;
+
+/**
+ * 
+ * @author andreas muelder - Initial contribution and API
+ * 
+ */
+public class SRuntimeFactoryImplOverride extends SRuntimeFactoryImpl implements SRuntimeFactory {
+
+	@Override
+	public Object createJavaObjectFromString(EDataType eDataType, String initialValue) {
+		int indexOf = initialValue.indexOf(":");
+		String className = initialValue.substring(0, indexOf);
```

Comment:
Are we sure that, `indexOf > 0` always holds? If not, an exception is thrown here. I would be more defensive.